keyboards/arch_36/keymaps/obosob/rules.mk

```diff
@@ -0,0 +1,17 @@
+BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration
```

Comment:
Please move this to the keyboard-level rules.mk.

Reply:
Shouldn't it use defaults for this generally, this is space saving for my keymap more than anything.

Reply:
I mean the entire file, minus your keymap-specific modifications. The keyboard-level rules.mk should have [this template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/rules.mk), and you can just override individual rules in this one.

Reply:
Gotcha, thanks,